contracts/instruments/RibbonThetaVault.sol

```diff
@@ -27,6 +28,7 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
     string private constant _adapterName = "OPYN_GAMMA";
 
     IProtocolAdapter public immutable adapter;
+    IVaultRegistry public immutable IRegistry;
```

Comment:
Normally for storage variables we use lowercases. We exclude the prefix 'I' because that's for interfaces. When you're instantiating an interface, `registry` should do